src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -68,7 +68,16 @@
       (compojure/GET "/services" []
         (let [statuses (call-status-fns status-fns-atom)]
           {:status 200
-           :body {"services" statuses}})))))
+           :body statuses}))
+       (compojure/GET "/services/:service-name" [service-name]
+         (if-let [status-info (get (deref status-fns-atom) service-name)]
+           (let [status (call-latest-status-fn-for-service status-info)]
+             {:status 200
+              :body (assoc status :service-name service-name)})
+           {:status 404
+            :body {:error {:type ::service-not-found
```

Comment:
https://github.com/rlinehan/pe-file-sync/commit/fff3b910900f14452a6faba23042f73e3b859849#commitcomment-10500725 and https://github.com/rlinehan/pe-file-sync/commit/dfc313bf473b57e1dd3000d5c5ebf44ed0bb70e9, although in that case I was kind of making up a partial namespace. I think sometimes having the response indicate what trapperkeeper service an error came from could maybe be useful in the case of http responses (e.g. `:status-service/service-not-found`), but I don't like having the full namespace in the response - I'd prefer just `:service-not-found` rather than the full namespace.